### PR TITLE
Fix for #3415: Available packages missing description

### PR DESCRIPTION
**Pull request by AlexanderSher**

### CODE REVIEW

File: src/Host/Client/Impl/rtvs/R/packages.R [hunk 1]
```diff
@@ -70,6 +70,7 @@ add.details <- function(repo.available, repo.url) {
     tryCatch({
         pkg.rds.path <- download.packages.rds(repo.url)
         repo.details <- read.packages.rds(pkg.rds.path)
+		repo.details <- repo.details[!duplicated(repo.details$Package),]
```
Comment (huguesv) [inline src/Host/Client/Impl/rtvs/R/packages.R]: Inconsistent indentation

**Comment by huguesv** [comment]
👍

**Comment by AlexanderSher** [comment]
Ok, one more review required

**Comment by MikhailArkhipov** [comment]
👍 

While you are there, could you fix https://github.com/Microsoft/RTVS/issues/3416, should trivial flip.

**Comment by AlexanderSher** [comment]
Yes, I'll fix it with #3424

**Pull request merged**